docs/source/getting-started/tabular-data/keys.ipynb

```diff
@@ -0,0 +1,280 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sum_year_actual = (df.group_by(\"YEAR\").agg(pl.col(\"HWUSUAL\").sum().alias(\"Actual Sum\")).sort(\"YEAR\")).collect()\n",
+    "\n",
+    "sum_year_dp = (\n",
+    "    context.query().group_by(\"YEAR\").agg(\n",
+    "        pl.col(\"HWUSUAL\").fill_null(40.).dp.sum((0,100)).alias(\"DP Sum\")\n",
+    "    ).sort(\"YEAR\")).release().collect()\n",
+    "\n",
+    "plt.figure(figsize=(10, 6))\n",
+    "sns.lineplot(x=sum_year_actual[\"YEAR\"].to_list(), y=sum_year_actual[\"Actual Sum\"].to_list(), marker=\"o\", label=\"Actual\")\n",
+    "sns.lineplot(x=sum_year_dp[\"YEAR\"].to_list(), y=sum_year_dp[\"DP Sum\"].to_list(), marker=\"o\", label=\"DP: Private\")\n",
+    "plt.title('Actual vs DP Sums by Year')\n",
+    "plt.xlabel('Year')\n",
+    "plt.ylabel('Sum')\n",
+    "plt.legend(title='Legend')\n",
+    "plt.show()"
```

Comment:
Did we ever consider using the polars hvplot integration? https://docs.pola.rs/api/python/stable/reference/dataframe/plot.html

With error bars, it may not help.

Reply:
yeah, but it had an unstable banner at the time and still does: `This functionality is currently considered unstable. It may be changed at any point without it being considered a breaking change.`